SteamKit2/SteamKit2/Util/LzmaUtil.cs

```diff
@@ -0,0 +1,125 @@
+﻿using System;
+using System.Buffers.Binary;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+
+namespace SteamKit2.Internal
+{
+    /// <summary>
+    /// Internal helper to decode data blobs with the 'LZMA' magic header.
+    /// </summary>
+    public static class LzmaUtil
```

Comment:
I would avoid exposing this publicly.

Reply:
I'd need InternalsVisibleTo for NHA2 then, or to move the whole thing into NHA2.

If GC are using it at the protocol level somehow then it would be needed by SK2 clients.
If they're not and its game-specific then I'd be on the fence about including it.

Reply:
Just put it in NH.

Reply:
Then I need a second copy of the LZMA SDK to get at the `Decoder`. 😞

Reply:
That's annoying. InternalsVisibleTo then? :(

Reply:
yeah, seems like the least evil option for now